wecSimTest.m

```diff
@@ -1,3 +1,51 @@
+function results = wecSimTest(options)
+
+    arguments
+        options.bemioTest = true
+        options.regressionTest = true
+        options.passiveYawTest = false
+        options.compilationTest = true
+        options.runFromSimTest = true
+        options.rotationTest = true
+    end
+    
+    import matlab.unittest.TestSuite;
```

Comment:
No need for semicolons on import functions. See [here](https://uk.mathworks.com/help/matlab/ref/import.html)